cli/src/pcluster/schemas/validators.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# This module contains all the classes representing the Schema of the configuration file.
+# These classes are created by following marshmallow syntax.
+#
+from pcluster.config.validators import FSX_MESSAGES
+from pcluster.constants import FSX_HDD_THROUGHPUT, FSX_SSD_THROUGHPUT
+
+
+def efs_existence_of_mode_throughput_validator(throughput_mode, provisioned_throughput):
```

Comment:
This code seems simple enough to reside in the EFSSchema

Reply:
Done